README.md

```diff
@@ -6,5 +6,16 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+Rails is a framework used to build the back end of a web app. It's purpose is to handle requests to create, read, upload, and delete stored data in the database.
+
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
```

Comment:
Correct! Though I will add, as a side note, that although it's uncommon for this to happen, controllers can in fact talk to _any_ model, or even none at all. The role of the controller is to respond to a type of request, and sometimes that request doesn't involve using the database in any way.